pkg/backend/httpstate/client/client.go

```diff
@@ -40,6 +40,8 @@ import (
 	"github.com/pulumi/pulumi/pkg/workspace"
 )
 
+const defaultPolicyGroup = "default-policy-group"
```

Comment:
Since this is used to refer to the default policy group, should it be exported from the `apitype` package instead? i.e. it's part of the ~Pulumi API SDK?